### ISSUE TITLE
Feature Request: A wp-cli command similar to scaffold plugin-tests

### ISSUE THREAD

**Issue by sudar** [10up/wp_mock#114]
This will make it easy for people to setup `wp_mock` for plugins and will also reduce the barrier to entry.

**Comment by ericmann** [comment]
I was thinking about this the other day in the context of #49, actually.

Do you think a WP CLI script would be best or perhaps a [Composer routine](https://getcomposer.org/doc/03-cli.md#create-project) that sets things up _both_ for WP_Mock and for the core test framework? Or maybe both?

**Comment by sudar** [comment]
I feel that a WP CLI script would be best, especially if it can set things up for both WP_Mock and core test framework.

The main reasons why I feel this way is because a) Core tests already use it (one less thing to learn) b) We need only the initial scaffolding.

**Comment by ericmann** [comment]
That works for me. The question the becomes, how do you set it up?

Would this need to be a separate package that you install globally so WP CLI can bootstrap things? I don't thing it would necessarily need to be a part of WP_Mock itself ...

**Comment by sudar** [comment]
Yes, having a seperate repo that you can install as a package in wp-cli makes most sense. It would be better to seperate WP_Mock and the WP Cli package repo.

**Comment by sudar** [comment]
@ericmann If you already have a repo where you are working on the wp-cli command then please let me. I am also willing to contribute to that repo.
